### DISCUSSION TITLE
Viewing camera/exif data in single photo view

### DISCUSSION THREAD

**Discussion by blalor** [photoprism/photoprism#2027] [Q&A] [answered]
I can't find a way to see the camera details and other metadata when viewing a single photo.  I can click into the edit page, but I'd like it displayed in a fairly compact form alongside the image. I feel like a small map view would be in the same place. Is this possible today and I'm not seeing it?

**Comment by graciousgrey** [comment, marked as answer]
You mean in fullscreen mode?
That's not possible with the current photo viewer. 
A customizable sidebar with metadata could be added after we implemented our own photo viewer (https://github.com/photoprism/photoprism/issues/1307)

**Reply by ghost** [reply]
this would be a great feature.

**Comment by blalor** [comment]
Yep, that’s it exactly! Thank you!

**Reply by Rathna-K** [reply]
looks like this is still not possible. any update or is this in the roadmap to show info of photo when viewed in slideshow or full screen mode?

**Reply by graciousgrey** [reply]
It will be part of the following issue: #1307 

See also question 2 in our FAQ: https://docs.photoprism.app/user-guide/faq/ :)
